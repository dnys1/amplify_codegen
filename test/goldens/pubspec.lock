# Generated by pub
# See https://dart.dev/tools/pub/glossary#lockfile
packages:
  amplify_core:
    dependency: transitive
    description:
      name: amplify_core
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.3.0-rc.3"
  amplify_datastore_plugin_interface:
    dependency: "direct main"
    description:
      name: amplify_datastore_plugin_interface
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.3.0-rc.3"
  amplify_lints:
    dependency: "direct dev"
    description:
      name: amplify_lints
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.0"
  characters:
    dependency: transitive
    description:
      name: characters
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.2.0"
  collection:
    dependency: transitive
    description:
      name: collection
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.15.0"
  crypto:
    dependency: transitive
    description:
      name: crypto
      url: "https://pub.dartlang.org"
    source: hosted
    version: "3.0.1"
  date_time_format:
    dependency: transitive
    description:
      name: date_time_format
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.0.1"
  flutter:
    dependency: "direct main"
    description: flutter
    source: sdk
    version: "0.0.0"
  flutter_lints:
    dependency: transitive
    description:
      name: flutter_lints
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.4"
  lints:
    dependency: "direct dev"
    description:
      name: lints
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.0.1"
  meta:
    dependency: transitive
    description:
      name: meta
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.7.0"
  plugin_platform_interface:
    dependency: transitive
    description:
      name: plugin_platform_interface
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.0.2"
  sky_engine:
    dependency: transitive
    description: flutter
    source: sdk
    version: "0.0.99"
  typed_data:
    dependency: transitive
    description:
      name: typed_data
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.3.0"
  uuid:
    dependency: transitive
    description:
      name: uuid
      url: "https://pub.dartlang.org"
    source: hosted
    version: "3.0.5"
  vector_math:
    dependency: transitive
    description:
      name: vector_math
      url: "https://pub.dartlang.org"
    source: hosted
    version: "2.1.1"
sdks:
  dart: ">=2.15.0 <3.0.0"
  flutter: ">=1.17.0"
