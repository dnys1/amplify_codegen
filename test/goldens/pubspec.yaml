name: goldens
description: Goldens for amplify_codegen
version: 1.0.0
publish_to: none

environment:
  sdk: '>=2.15.0 <3.0.0'

dependencies:
  amplify_datastore_plugin_interface: ^0.3.0-0
  flutter:
    sdk: flutter

dev_dependencies:
  amplify_lints: ^1.0.0
  lints: ^1.0.0
